README.md

```diff
@@ -0,0 +1,48 @@
+# NC News
+
+This project is a front-end web application which mimics a site such as Reddit, using an API that was built for a backend project.
+
+**Hosted app:**
+
+https://ncsubsidiarynews.netlify.app/
+
+**Backend repo:**
+
+https://github.com/DeanDevine/be-nc-news
+
+**Hosted backend database:**
+
+https://be-nc-news-whiy.onrender.com/api/
+
+# Instructions on using the app:
+
+Users can currently:
+
+* view articles - these can be filtered by topic and sorted with * queries
+* like/dislike articles
+* comment on articles and remove their comments
+
+To comment on an article a user must first sign in on a valid username. Valid usernames are:
+
+* tickle122
+* grumpy19
+* happyamy2016
+* cooljmessy
+* weegembump
+* jessjelly
+
+# Instructions on running this project locally:
+
+**Clone the repository by running the following command in the terminal:**
+
+git clone https://github.com/DeanDevine/nc-news.git
+
+**Install axios, react, react-dom, react-router-dom and vite package dependencies with the following command in the terminal:**
```

Comment:
Not sure you need to list these, just tell them to run npm install - as you have below